### PR TITLE
style: replace broken glyphs with font-awesome

### PR DESCRIPTION
**Pull request by mistercrunch**
Glyphicons stopped working recently, not sure why, but let's get rid of
them and double down on font-awesome that we use a lot more in the
codebase. There's only a few instances of glyphicons and they all are
broken ATM.

Also a few other minor style tweaks

## before
Screen Shot 2020-06-20 at 11 03 51 PM
Screen Shot 2020-06-20 at 11 03 28 PM

## after

Screen Shot 2020-06-20 at 10 58 58 PM
Screen Shot 2020-06-20 at 10 44 58 PM
Screen Shot 2020-06-20 at 10 42 56 PM
Screen Shot 2020-06-20 at 10 42 35 PM

### CODE REVIEW

File: superset-frontend/src/explore/components/AdhocMetricOption.jsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ export default class AdhocMetricOption extends React.PureComponent {
     // once the overlay has been opened, the metric/filter will never be
     // considered new again.
     this.props.adhocMetric.isNew = false;
-    this.setState({ overlayShown: false });
+    this.setState({ overlayShown: true });
```
Comment (etr2460) [inline superset-frontend/src/explore/components/AdhocMetricOption.jsx:56]: Is this related to the glyph changes? is it intentional?

**Review by etr2460 — COMMENTED** [review]
one question, otherwise lgtm.

I think we'll need to go through the chart plugins and fix these too, since i know tables have been broken because of this.

cc @michellethomas who was tracking this bug

**Review by etr2460 — APPROVED** [review]

**Comment by ktmud** [comment]
There's a new table chart coming soon. Hopefully we don't have to have another fix once [the drop-in replacement](https://github.com/apache-superset/superset-ui/pull/623) is merged.

P.S. I was using [react-icons](https://react-icons.github.io/react-icons/) for the new table chart. With tree shaking properly configured, these svg icons packs should work very well.

File: superset-frontend/src/explore/components/AdhocMetricOption.jsx (hunk 1 above)
Comment (mistercrunch) [reply superset-frontend/src/explore/components/AdhocMetricOption.jsx:56]: That was a minor bug I uncovered, as I change the icons to `caret-(left|right)`

**Comment by mistercrunch** [comment]
@ktmud `SIP-34` provided icons as SVG, they're starting to land. Consistency would be great eventually. Between glyphicons, SIP-34 svg, react-icons, font-awesome, ... Maybe we need a `superset-ui-icons` package :)

@etr2460 it'd be nice to be able to `grep` across all superset code. I `git grep` a lot and totally miss out on other repos. Also noticed that the documented `i18n` command currently doesn't inventorize the strings in `superset-ui*`, I spent a bit of time trying to fix this messing with `babel.cfg` (the python i18n babe-thing, not the JS one), with symlink, but nothing worked. Now I'm thinking git submodules OR a big mono-repo may work.

**Comment by etr2460** [comment]
I agree @mistercrunch, not having i18n work for superset-ui is a real pain :/ That said, getting all the strings working/fixed in the main repo is enough of a task, not even considering the side repos. There are a lot of mistaken usages of `t` strings that i've been meaning to clean up
